egs/wsj/s5/utils/lang/bpe/apply_bpe.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, codes, merges=-1, separator='@@', vocab=None, glossaries=None
 
         self.vocab = vocab
 
-        self.glossaries = glossaries if glossaries else []
+        self.glossaries = " ".join(glossaries).split() if glossaries else []
```

Comment:
why is this necessary? when would glossaries be different from " ".join(glossaries).split()?

Reply:
You are right. The changes here is unnecessary.